gh-155 Improve pagination control on smaller screen widths

pjmonks opened the pull request:
- Reduce number of page numbers to display
- Use flexbox container to forces select and page controls to wrap when screen width is too small
- Remove total number of pages count from pagination component, seems superfluous

Fixes #155 

# Screenshots

Large screen width:

![image](https://user-images.githubusercontent.com/3219480/208889029-839a1c84-67e2-48fa-b193-0ccafaa2b578.png)

Small screen width:

![image](https://user-images.githubusercontent.com/3219480/208888968-8d2b05f2-2b67-4650-be07-3016255d3544.png)

NigelPalmer reviewed (commented):
- [X] Page resizing works without controls overlaying each other.
- [X] Pages are now limited to 5 being displayed.
- [ ] I disagree that the total page number count is superfluous. When I had 100 items, there were 10 pages, but the page indicator topped out at 5, so it was not clear that there are more than 5 pages. The way I am used to this information being presented is: 
![image](https://user-images.githubusercontent.com/110250803/210973333-7a8fe548-4d58-40ad-af91-80cdb5687d14.png) Could we adapt the pagination control to work in such a way?

pjmonks commented:
I've changed the pagination control to include the page counter again at least, updated as well to show this:

![image](https://user-images.githubusercontent.com/3219480/211535406-c6282839-615a-4d26-9516-aa5a008ece13.png)

This is the best I can do on narrow screen widths, but the expectation is this probably won't be viewed via mobile for the foreseeable future.

NigelPalmer reviewed (approved):
- [X] Commented out code removed.
- [X] Page count restored.
**Passed**
